src/OPUBL-1.0.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="OPUBL-1.0" isOsiApproved="false" name="Open Publication License v1.0" listVersionAdded="3.14">
+    <crossRefs>
+      <crossRef>http://opencontent.org/openpub/</crossRef>
+      <crossRef>https://www.debian.org/opl</crossRef>
+      <crossRef>https://www.ctan.org/license/opl</crossRef>
+    </crossRefs>
+    <notes>Users of this license may wish to take care to identify which, if any, of the 'license options' described in section VI apply to the licensed work. The Free Software Foundation considers only the form with no license options applied to be a free documentation license.</notes>
+    <text>
```

Comment:
we should remove the sentence about what the FSF thinks - we don't ever include that kind of thing as it's a interpretation.

Reply:
Right'o - patch added to this pull request.